### DISCUSSION TITLE
Docker container

### DISCUSSION THREAD

**Discussion by GitFun4All** [time4tea/gopro-dashboard-overlay#72] [Ideas]
Hi,

Firstly, thanks for gorps-dashboard-overlay - I'm using it with my Hero8 generated files and so far it works perfectly!

Setting up the components was a bit of a pain so I've created some docker resources to do everything required.

Maybe you could include something similar here directly?

**Comment by time4tea** [comment]
That's a great idea! - Thanks! - I'll create an issue for it.

**Comment by time4tea** [comment]
I created: https://github.com/time4tea/gopro-dashboard-overlay/issues/73

**Comment by GitFun4All** [comment]
I have a rough outline already, but it needs some polishing. Might be a good start point : https://github.com/GitFun4All/GoPro-dash

**Comment by time4tea** [comment]
This is awesome - Would you mind if i incorporated some of this into this repo? The code for gopro-dashboard-overlay is licensed under the MIT license - so just would like you to confirm that you are happy with that....

**Reply by GitFun4All** [reply]
Sure, no problem (I hope it works for you too!).
